src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -667,6 +697,9 @@ private static boolean isLoopAst(int ast) {
         /** Contains definitions of uninitialized variables. */
         private final Deque<DetailAST> uninitializedVariables = new ArrayDeque<>();
 
+        /** Whether there is a {@code break} or {@code return} in the scope. */
+        private boolean containsBreakOrReturn;
```

Comment:
> > ... but false negatives wouldn't show there.

> Maybe I don't understand clearly... Is there anything unexpected in the diff report?

Nothing in the diff report.
I was merely stating the diff report only shows us violations, removed or added. It doesn't tell us cases where we are still missing out on variables that should get a violation.